packages/typescript/src/lib/Typescript.ts

```diff
@@ -13,6 +13,7 @@ export class FederatedTypesPlugin {
   private exposedComponents!: ModuleFederationPluginOptions['exposes'];
   private remoteComponents!: ModuleFederationPluginOptions['remotes'];
   private distDir!: string;
+  private fsCache: Map<string, string[]>;
```

Comment:
NIT: 
```suggestion
  private fsCache: Map<string, string[]> = new Map();
```

with this we don't have to assign it in the constructor.